lib/happo/server.rb

```diff
@@ -43,14 +43,18 @@ def h(text)
       file = params[:splat].first
       if File.exist?(file)
         send_file file
+        return
       else
```

Comment:
This is a minor issue, but since we now return in the if-statement above, we don't need the `else`.